grafana-plugin/src/pages/settings/SettingsPage.tsx

```diff
@@ -41,7 +37,7 @@ class Settings extends React.Component<SettingsPageProps, SettingsPageState> {
   }
 
   render() {
-    return <div className={cx('root')}>{this.renderContent()}</div>;
+    return <div>{this.renderContent()}</div>;
```

Comment:
do we need `div` here?

Reply:
This is existing structure, I'm not changing how the layout was done

Reply:
I'm almost sure this div was added just to hold `root` class and nothing else, but you are right is not the scope of your task